src/components/events/CalendarButtons/style.module.scss

```diff
@@ -0,0 +1,35 @@
+.options {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+}
+
+.calendarButton {
+  align-items: center;
+  background: transparent;
+  border: 1px solid var(--theme-primary-2);
+  border-radius: 10px;
+  display: flex;
+  gap: 10px;
+  justify-content: center;
```

Comment:
Maybe it should be left aligned so the icons and "Add to" lies up?

```suggestion
  justify-content: flex-start;
```

though for some reason the icons don't seem to be properly aligned (the Google logo is off center)

![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/41d35c1e-7eb2-4b72-b897-87752d6c7fe8)

Reply:
I've updated the logos since there was a weird gap in the Google logo, and fixed the alignment